src/test/java/com/openlattice/kindling/search/BaseElasticsearchTest.java

```diff
@@ -179,7 +180,7 @@ public static void initEdmObjects() {
                 Optional.of( "employees that are in chicago" ),
                 ImmutableSet.of( "foo@bar.com", "foobar@foo.net" ),
                 Optional.empty(),
-                Optional.empty(),
+                IdConstants.GLOBAL_ORGANIZATION_ID.getId(),
```

Comment:
🤔

Reply:
ehh i just used that since it was the default before but could switch it to random